.github/workflows/main.yml

```diff
@@ -140,13 +134,20 @@ jobs:
               if all(pkg not in line for pkg in excluded_on_win):
                   print(line)
 
+      - name: Setup mini-conda
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: snakemake
+          python-version: 3.8
+          channels: conda-forge, bioconda
+
       - name: Setup Snakemake environment
         run: |
-          conda install -c conda-forge -q mamba
-          mamba env create -q --name snakemake --file test-environment.yml
-        
+          conda install -q --name base mamba
+          mamba env update -q --file test-environment.yml
```

Comment:
Mhm, why putting everything into base instead of creating a separate environment?

Reply:
It does goes into the snakemake environment. But I agree that it is a little hard to spot. 

I had a lot trouble getting the conda working properly. I ended up using the ["setup-miniconda" gihtub action](https://github.com/marketplace/actions/setup-miniconda). It creates and activates the snakemake environment for all subsequent steps. So it is only mamba that goes into the base environment. The `mamba env update ..` line works on the snakemake environment. 

I hope the action will soon support mamba and then this should become nicer.